### ISSUE TITLE
[QUESTION] Does Imagus Mod 0.10.15 for Chrome uninstall itself?

### ISSUE THREAD

**Issue by gaveitatry2** [TheFantasticWarrior/chrome-extension-imagus#73]
I thought that I already installed this, but today I noticed that it was missing so I reinstalled it.

Does this mod uninstall itself after a Chrome update?

**Comment by kuzn123** [comment]
"Download Imagus Mod 0.10.15 in ZIP, unzip it to your browser folder (for example, [like this](https://i.imgur.com/xvaK0F8.png)), then Settings - Manage Extensions - enable "Developer Mode" (if disabled) - "Load unpacked".

If Chrome or Chromium browser writes that the extension is "installed with errors" - this is normal".
([FAQ](http://forum.ru-board.com/topic.cgi?forum=5&topic=50874&start=0&limit=1&m=7#1), p.19)

**Comment by gaveitatry2** [comment]
> "Download Imagus Mod 0.10.15 in ZIP, unzip it to your browser folder (for example, [like this](https://i.imgur.com/xvaK0F8.png)), then Settings - Manage Extensions - enable "Developer Mode" (if disabled) - "Load unpacked".
> 
> If Chrome or Chromium browser writes that the extension is "installed with errors" - this is normal". ([FAQ](http://forum.ru-board.com/topic.cgi?forum=5&topic=50874&start=0&limit=1&m=7#1), p.19)

Yes. I know all that. That is exactly what I did. Yesterday, I had to reinstall it because it was missing. I installed it following those same exact steps. I saw the error message but I ignored it. Imagus worked on all the websites that I had whitelisted. But today, this morning, I looked and the Imagus extension was missing again. It seems to uninstall itself for no reason.

**Comment by gaveitatry2** [comment]
I don't know if this will help... but I found a Reddit thread with a post from one year ago where a guy says that the unpacked Imagus Mod needs to be a permanent directory. It can't be deleted. So this time I unpacked to my C Drive into a Programs folder and I didn't delete it after I installed the extension. Hopefully this time it will stay. If not, then I need to figure out how to get to the CRX folder/files (have no idea how) as explained in Step 19 in the FAQ.

**Comment by kuzn123** [comment]
>> Yes. I know all that. That is exactly what I did. Yesterday, I had to reinstall it because it was missing. I installed it following those same exact steps.

Are you sure you did it this way? The thing is that the standard installation of the extension via unzipping **installs Imagus Mod in the TEMP directory - which automatically leads to its removal**. That's why we recommend creating an IM directory in the program's (Chrome) folder. Maybe you should read my post above carefully again and try to install Imagus as recommended?

>> how to get to the CRX folder/files

What do you mean? All you need to do is download the archive [IM 0.10.15](https://github.com/TheFantasticWarrior/chrome-extension-imagus/releases/download/v0.10.15/imagus_mod-0.10.15-chrome.zip) and unzip it into your Chrome program directory. And then go to "Manage Extension" in Settings, turn on "Developer Mode" (when turned off), click "Load Unpacked" and select the directory with IM in the Chrome directory > OK. After this sequence of actions, the directory with IM does not disappear from Chrome, at least no one has complained about it.

**Comment by gaveitatry2** [comment]
"Maybe you should read my post above carefully again and try to install Imagus as recommended?"

Why don't you try removing the stick from your butt and stop intellectually insulting people?

Anyway, you are right. I did not "unzip it to your browser folder". I was unzipping it to my E drive and then after I did the "Load Unpacked" and later deleted the unzipped folder, the extension was also deleted. It doesn't matter WHERE a person unzips the folder. It doesn't HAVE to be in the browser folder. The thing is we are supposed to NOT DELETE the unzipped folder after we are done. And I didn't see that in the FAQ (maybe I missed it). I have since unzipped it to a Programs folder on my C Drive, and so far the extension hasn't been deleted, probably because I never deleted the unzipped folder.

But yeah, I don't like how rude you are to people. Acting all uppity and stroking your own ego while talking down to others. You should work on that.

**Comment by TheFantasticWarrior** [comment]
I just tested and chrome update doesn't remove the extension, however on older versions of chrome/other reasons it would install in temp folder, when I tried to install on latest chrome version it installed in what seems like a permanent folder (the steps are the same, but it still could be other reasons than chrome version)

I don't think it is related to deleting the folder before the load unpacked step, but I have no idea what's going on, so...

Also, sorry for the behavior from @kuzn123 I added him to answer questions for me as he seems to be the person that answers questions on the original extension. 
I think he uses translation from Russian and doesn't know that it's insulting, and answering the same questions again and again is exhausting, which is an understandable reason but still doesn't make it ok. (The part you quoted actually sounds ok to me but I am bad at those things, I do know that his attitude is bad sometimes)

@kuzn123 I forgot if I told you or not but please be nicer to people

**Comment by kuzn123** [comment]
>> I forgot if I told you or not but please be nicer to people

I have not insulted this user anywhere or in any way. If you saw something “offensive” in my post - give me a quote. If you can't provide a quote where I directly insulted the user - then please don't give me advice on how to behave with users. However, I realize that this is my fault. I have already promised myself once that I will not interfere in discussions on this resource. And... I couldn't resist doing it;) I hope that was the last time.

**Comment by gaveitatry2** [comment]
"I don't think it is related to deleting the folder before the load unpacked step"

For me, I was deleting the zip folder after (why did you say before?) I did the load unpacked step, and this caused the extension to be removed. So I permanently keep the zip folder somewhere on one of my drives now, and now the extension stays. It's been 2-3 days, and my extension is still there. Hopefully it will be there after a Chrome update. I'm on the latest version of Chrome.

**Comment by gaveitatry2** [comment]
Closing this because I learned that extensions dont' delete themselves unless you delete the zip folder that you used to install them.
